### PR TITLE
Fix unresolved name issue

### PR DESCRIPTION
**Pull request by adamtheturtle**
This was found with `pyflakes`. I'd be happy to add a `pyflakes` checker to the Travis config if that is desired.

### CODE REVIEW

File: src/txkube/testing/_eliot.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 from fixtures import Fixture
 
-from eliot import add_destination
+from eliot import add_destination, remove_destination
```
Comment (adamtheturtle) [inline src/txkube/testing/_eliot.py:8]: See https://github.com/LeastAuthority/txkube/pull/8/files#diff-58b820f05af5c1f18e36eef9bede77cbR64 (the end of this file) for the use of this.

**Comment by exarkun** [comment]
> I'd be happy to add a pyflakes checker to the Travis config if that is desired.

@adamtheturtle Yea, that'd be awesome.  It's on my todo list but happy for someone else to get there first.  Feel free to stick some of those other static checks you mentioned elsewhere in, if they seem appropriate.

File: src/txkube/testing/_eliot.py (hunk 1 above)
Comment (exarkun) [reply src/txkube/testing/_eliot.py:8]: Yep.  This is only half the fix, as it turns out.  I noticed the problem at some point during development of #5 - there is actually no `Fixture._tearDown`  :(  Instead, the code should `self.addCleanup(...)` inside `_setUp`.  Feel like doing the rest of that here?

**Review by exarkun — CHANGES_REQUESTED** [review]

**Comment by adamtheturtle** [comment]
Yes I'll get to the other 😄

**Pull request merged**